### ISSUE TITLE
Feature request: Possibility to exclude items marked as deleted

### ISSUE THREAD

**Issue by Boulder08** [Lazza/RecuperaBit#36]
I have a 3TB drive that I'm currently recovering files from. I can get the list of files nice and easy with the analysis pass so it would be easy to just use restore x 5 and restore x -1 to recover everything. However, it seems that some previously deleted files (marked as deleted in the csv output) are being recovered as well, and this means that I run out of disk space if I use the simple route (I don't have a 4TB drive to recover to).

In my case, I don't need those deleted files as they are either from the Windows Recycle Bin or directly deleted by me or some program. It would be nice if I could use a parameter to skip the deleted files or maybe skip files and/or directories by name using a wildcard.

**Comment by Lazza** [comment]
Yes, as I have discussed before the CLI interface needs a revamp. It is currently basic and a bit messy while the reconstruction code is pretty solid and tested.

In the meanwhile you might want to add an extra `if` condition to `recursive_restore` which is inside `logic.py`. Something like this as the first line of the function should do what you want:

    if node.is_ghost: return
